### DISCUSSION TITLE
h o w d y, Biché

### DISCUSSION THREAD

**Discussion by extratone** [extratone/bilge#51] [ＨＯＬＥ]
## Trying out Discussions.

[An Important Tweet](https://twitter.com/NeoYokel/status/1336374708753797123)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edrifmohamed11** [comment]
Que oculta el área 51
